src/number_theory/function_field.lean

```diff
@@ -14,7 +14,7 @@ This file defines a function field and the ring of integers corresponding to it.
 
 ## Main definitions
  - `function_field Fq F` states that `F` is a function field over the (finite) field `Fq`,
-   i.e. it is a finite extension of the field of rational polynomials in one variable over `Fq`.
+   i.e. it is a finite extension of the field of rational funcions in one variable over `Fq`.
```

Comment:
```suggestion
   i.e. it is a finite extension of the field of rational functions in one variable over `Fq`.
```
Also elsewhere.

Reply:
A search-and-replace error, fixed with some more searching-and-replacing!